Serializer/Denormalizer/RouteCollectionDenormalizer.php

```diff
@@ -19,9 +19,8 @@ class RouteCollectionDenormalizer implements DenormalizerInterface
 {
     /**
      * {@inheritDoc}
-     * @return mixed
      */
-    public function denormalize($data, $class, $format = null, array $context = array())
+    public function denormalize($data, $type, $format = null, array $context = []): RouteCollection
```

Comment:
This might be a BC break. I think we should remove the return type?

```suggestion
    public function denormalize($data, $type, $format = null, array $context = [])
```